lib/pxconfig.js

```diff
@@ -30,6 +30,17 @@ class PxConfig {
         this.configLoader = null;
     }
 
+    get remoteConfigVersion() {
+        return this.config.px_remote_config_version || INVALID_VERSION_NUMBER;
+    }
+
+    get isRemoteConfigOutdated() {
```

Comment:
I don't think this belongs here, the metadata that indicates if this instance is s need to be updated should be handled outside of it.
See elaborate suggestion below.

Reply:
Moved to pxClient